locales/en-US/notes.properties

```diff
@@ -16,3 +16,12 @@ disableSync=Disable Sync
 # structure doesn't work for your locale, you can translate this as "Last sync:
 # {date}".
 syncComplete=Synced {date}
+
+# Titles for toolbar buttons
```

Comment:
Changes made, thank you for letting me do this!  I find it very fun and rewarding being able to contribute. 🙌 

Would definitely like to tackle other issues if you don't mind.